cmd/create_package.go

```diff
@@ -81,10 +82,19 @@ func createPackageCommandAction(cmd *cobra.Command, args []string) error {
 			Name: "kibana_version",
 			Prompt: &survey.Input{
 				Message: "Kibana version constraint:",
-				Default: surveyext.DefaultConstraintValue(),
+				Default: surveyext.DefaultKibanaVersionConditionValue(),
 			},
 			Validate: survey.ComposeValidators(survey.Required, surveyext.ConstraintValidator),
 		},
+		{
+			Name: "elastic_subscription",
+			Prompt: &survey.Select{
+				Message: "Required Elastic subscription:",
+				Options: []string{"basic", "gold", "platinum", "enterprise"},
```

Comment:
nit: I would change the order to be shown like [here](https://www.elastic.co/subscriptions)

```suggestion
				Options: []string{"basic", "platinum", "enterprise", "gold"},
```

Reply:
Interesting, I didn't know that gold is discontinued, I guess that this is the only reason why it appears later there. But technically platinum > gold, we have been using this order everywhere. See the spec for example: https://github.com/elastic/package-spec/blob/0456e4f000a5bf85130d4e5e9c6d329a627523ec/versions/1/integration/manifest.spec.yml#L54